background/get-addon-settings.js

```diff
@@ -322,6 +333,110 @@ chrome.storage.sync.get([...ADDON_SETTINGS_KEYS, "addonsEnabled"], (storageItems
             settings.popup = newPopupSettingValue;
             madeAnyChanges = madeChangesToAddon = true;
           }
+
+          updatePresetIfMatching(
+            settings,
+            7,
+            {
+              // "Experimental Dark" preset
+              page: "#263241",
+              primary: "#4d97ff",
+              highlightText: "#4d97ff",
+              menuBar: "#4d97ff",
+              activeTab: "#282828",
+              tab: "#202020",
+              selector: "#252c37",
+              selector2: "#202020",
+              selectorSelection: "#282828",
+              accent: "#282828",
+              input: "#282828",
+              workspace: "#282828",
+              categoryMenu: "#282828",
+              palette: "#333333cc",
+              border: "#444444",
+            },
+            () => {
+              console.log("Migrated Experimental Dark preset.");
+              madeAnyChanges = madeChangesToAddon = true;
+              settings.page = "#2e3238";
+              settings.primary = "#855cd6";
+              settings.highlightText = "#ccb3ff";
+              settings.menuBar = "#855cd6";
```

Comment:
Is this line necessary? The menu bar color is updated unconditionally.

Reply:
Yeah it's probably unnecessary but I think I got the color wrong in migration #10. Please see code comment below I just created.

Reply:
Yeah, this line is unnecessary then.

Reply:
Fixed by https://github.com/ScratchAddons/ScratchAddons/pull/6074/commits/487ed9c392bff7908c1837a5a24f4856bd5abe4e